cluster/addons/dashboard/OWNERS

```diff
@@ -1,16 +1,13 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-approvers:
-  - floreks
-  - maciaszczykm
-  - shu-mutou
-reviewers:
-  - floreks
-  - maciaszczykm
-  - shu-mutou
+approvers: []
```

Comment:
It would make it impossible to approve any change update to the dashboard addon (for everyone except repo admins). 
I see 2 options:
- revert this change
- remove the addon as deployment configuration is very outdated anyway and I am not sure if we can keep it up-to-date at all times

Reply:
@floreks when a OWNERS file in a specific directory is empty, the approver plugin looks in its parent directory (until it reaches root directory of the repo). So a repo admin is not needed

Reply:
I'd anyway vote for removing this addon entirely.

Reply:
@floreks yes please go ahead and file a PR to remove this addon!